src/ui/common/src/components/workflows/workflowHeader.tsx

```diff
@@ -41,11 +41,30 @@ type Props = {
   workflowId: string;
 };
 
+const ContainerWidthBreakpoint = 700;
+
 const WorkflowHeader: React.FC<Props> = ({ user, workflowDag, workflowId }) => {
   const dispatch: AppDispatch = useDispatch();
   const { apiAddress } = useAqueductConsts();
   const navigate = useNavigate();
 
+  const currentNode = useSelector(
+    (state: RootState) => state.nodeSelectionReducer.selected
+  );
+
+  // NOTE: The 1000 here is just a placeholder. By the time the page snaps into place,
+  // it will be overridden.
+  const [containerWidth, setContainerWidth] = useState(1000);
+  const narrowView = containerWidth < ContainerWidthBreakpoint;
+
+  const getContainerSize = () => {
```

Comment:
did you look into the useMediaQuery hook that MUI provides at all?
https://mui.com/material-ui/react-use-media-query/

Might be something we can use here.

Reply:
Yeah, I looked at this and some React libraries. I did quite a bit of Google/SO-ing, and I wasn't able to find any solutions that were simpler than just using `document.getElementByID`. Happy to entertain other ideas, but this felt like the most straightforward.